File: CS/ChartsDemo.Wpf/Utils/ChartAnimationService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.Xpf.Charts;
using DevExpress.Mvvm.Native;
using DevExpress.Mvvm.UI;
using DevExpress.Mvvm.UI.Interactivity;

namespace ChartsDemo {
    public class ChartAnimationService : ServiceBase, IChartAnimationService {
        ChartControl Chart { get { return (ChartControl)AssociatedObject; } }
        public void Animate() {
            Chart.Dispatcher.BeginInvoke(new Action(Chart.Animate));
        }
    }
}

File: VB/ChartsDemo.Wpf/Utils/ChartAnimationService.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.Xpf.Charts
Imports DevExpress.Mvvm.Native
Imports DevExpress.Mvvm.UI
Imports DevExpress.Mvvm.UI.Interactivity

Namespace ChartsDemo
    Public Class ChartAnimationService
        Inherits ServiceBase
        Implements IChartAnimationService

        Private ReadOnly Property Chart() As ChartControl
            Get
                Return CType(AssociatedObject, ChartControl)
            End Get
        End Property
        Public Sub Animate() Implements IChartAnimationService.Animate
            Chart.Dispatcher.BeginInvoke(New Action(AddressOf Chart.Animate))
        End Sub
    End Class
End Namespace
